### PR TITLE
MTA-523 Release Notes for 6.1.1

### PR DESCRIPTION
**Pull request by anarnold97**
First draft for [MTA 523](https://issues.redhat.com/browse/MTA-523)

### CODE REVIEW

File: docs/topics/mta-rn-resolved-issues-6-1-1.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-1_{context}"]
+= Resolved issues
+
+At the time of the release, the following resolved issue has been identified as the major issue worth highlighting:
+
+.RHSSO
+
+The release of {ProductShortName} 6.1.1 resolves the issue that users experienced after the the release of version 7.6.4 of Red Hat Single Sign-On (RH-SSO), which rendered many new installations of {ProductShortName} 6.x unusable.
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-6-1-1.adoc:13]: @rromannissen & @dymurray please can I ask if you are ok with this draft? I know it is a bit rough, but I am trying to ensure we are not a blocker.

I might change:

* {ProductShortName} *6.x* to *6.X.X*, or do you have a preferred format?
* Change *unusable* to *unstable*

Do you want the ticket number [RHSSO-2514](https://issues.redhat.com/browse/RHSSO-2514) including at the end of the issue?

I have amended the default wording to highlight the singe issue.

**Comment by anarnold97** [comment]
@rromannissen & @dymurray - please, can you do a review? 
Thanks

**Comment by anarnold97** [comment]
Do you want me to tidy up the following and add to 6.1.0 release notes as a *Known Issue*:

This issue will impact all MTA 6 customers, it will break existing installs and new installs.  MTA has a dependency on the RHSSO Operator, the RHSSO operator released an update yesterday with a bug:  [https://issues.redhat.com/browse/RHSSO-2514](https://www.google.com/url?q=https%3A%2F%2Fissues.redhat.com%2Fbrowse%2FRHSSO-2514&sa=D&sntz=1&usg=AOvVaw0_h2TV5JlLNdGQGAuAWHGb)

The bug runs a 'grep' in the "/opt/eap/bin/openshift-launch.sh" command from the keycloak pod.  The grep assumes it will only find a single match, it finds 2 matches and it breaks.  
 
Issue shows up as:
```
$ oc get pods -n openshift-mta
NAME                                       READY   STATUS             RESTARTS      AGE
keycloak-0                                 0/1     CrashLoopBackOff   6 (36s ago)   8m17s
mta-keycloak-postgresql-85b4949756-g2lb9   1/1     Running            0             9m25s
mta-operator-69b8d7db59-fbckt              1/1     Running            0             12m
rhsso-operator-7499fb4cd7-n977r            1/1     Running            0             12m

$ oc logs keycloak-0 -n openshift-mta
```


And include in release notes for 6.1.0

File: docs/topics/mta-rn-resolved-issues-6-1-1.adoc (hunk 1 above)
Comment (anarnold97) [reply docs/topics/mta-rn-resolved-issues-6-1-1.adoc]: Thanks @rromannissen

File: docs/topics/mta-rn-resolved-issues-6-1-1.adoc [hunk 2]
```diff
@@ -10,5 +10,5 @@ At the time of the release, the following resolved issue has been identified as
 
 .RHSSO
 
-The release of {ProductShortName} 6.1.1 resolves the issue that users experienced after the the release of version 7.6.4 of Red Hat Single Sign-On (RH-SSO), which rendered many new installations of {ProductShortName} 6.x unusable.
+The release of {ProductShortName} 6.1.1 resolves the issue that users experienced after the the release of version 7.6.4 of Red Hat Single Sign-On (RH-SSO), which rendered many new installations of {ProductShortName} 6.x unusable. For more information on this issue, see  link:https://issues.redhat.com/browse/RHSSO-2514[RHSSO-2514].
```
Comment (rromannissen) [inline docs/topics/mta-rn-resolved-issues-6-1-1.adoc:13]: after the release of 7.6.3

**Review by rromannissen — CHANGES_REQUESTED** [review]

**Review by rromannissen — APPROVED** [review]

**Comment by RichardHoch** [comment]
LGTM

**Pull request merged**